documentation/getting_started/making_mods/unity_modding/index.md

```diff
@@ -0,0 +1,115 @@
+﻿---
+title: Unity Modding setup
+permalink: /making_mods/unity_modding/
+nav_order: 0
+layout: page
+has_toc: true
+parent: Making Mods
+---
+# Unity Modding setup
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+## Common Problem:
+* Import window:  
+    When this window is working it may take some time.  
+    it may Crash and need to be Force closed and opened again and thats normal when changing version or adding alot of new assets or settings.  
+    meaning if its stuck importing on one package over 1-2 min then shut it down with taskmanager.
+
+## Find version of unity
+
+### Thunderkit
+To find the right version Find the Timberborn.exe and look at property's for it.   
+there Filevesion is show thats the version of Unity Thunderkit will need.   
+at writing 2022.1.6f1 is know to work
+
+### Mod unity version
+this will depend on what version timberAPI uses.  
+at writing 2021.3.1f1 is know to work
+
+
+## Instalation
+1. Download Zip [MinimalModSetup](https://github.com/KnatteAnka/MinimalModSetup)  
+1. Install [UnityHub](https://unity3d.com/get-unity/download)
+1. Open Unityhub/Installs and Click "Install Editor"
+1. Select Archive and click download archive and find the versions from above and install those
+
+## Create Project
+1. Unpack MinimalModSetup content and rename folder to your liking
+1. Open Unityhub/Projects and press Open and select folder from above
+
+## Thunderkit setup:
+
+### Steps to Import game Scripts
+1. Change version to use Thunderkit version. (Click on version in column and select the correct one)
+1. It will then Open the project, warning may show about version and just click continue
+
+1. Wait for Editor to open
+    * if question about Safe mode is shown select  
+    if it does press exit safe mode in the top left corner
+1. ThunderKit settings window should now show else its found:  
+    ![Thunderkit](/assets/images/assetripper/thunderkit.png)  
+    * When you have the thunderkit window open you can click on Show at startup to have it hidden as default.
+1. Select thunderkit settings and browse for the timberborn.exe and press import  
+![Thunderkit](/assets/images/assetripper/thunderkit_import.png)  
+    
+    
+
+## Modding Setup:   
+1. Close the project and change version to modding version as described above.
+1. It will then Open the project, warning may show about version and just click continue
+1. Wait for Editor to open, This may take some time
+    * If question about Safe mode is shown select  
```

Comment:
select what?

Reply:
This is now clarifyed:
1. Wait for Editor to open, This may take some time
    * If question about Safe mode is shown select "Enter Safe Mode".  
    * If it does press exit safe mode in the top left corner